docs/plugins/requirements/image-processing.md

```diff
@@ -93,6 +93,189 @@ The following environments come with a preinstalled version of [Cairo Graphics]:
   [Docker image]: https://hub.docker.com/r/squidfunk/mkdocs-material/
   [GitHub Actions]: ../../publishing-your-site.md#with-github-actions
 
+#### Troubleshooting { id="cairosvg-troubleshooting" }
```

Comment:
Why the anchor name? I'd say we'll keep it as troubleshooting to make room for more to add later + keep the anchor links stable. If we feel that we need more granularity, we can move troubleshooting one way up and add further headlines.

Reply:
OK, so I misunderstood, I'll move the Troubleshooting to the bottom of the page as another H2 tag, a sibling of Dependencies, and change the link to `#troubleshooting` 👌